src/simsopt/geo/surfacerzfourier.py

```diff
@@ -421,6 +421,75 @@ def from_pyQSC(cls, stel: Qsc, r: float = 0.1, ntheta=20, mpol=10, ntor=20, **kw
 
         surf.local_full_x = surf.get_dofs()
         return surf
+    
+    def copy(self, **kwargs): 
```

Comment:
What is the difference between this function and simply setting the dofs of a new surface object as, for example

`
surf = SurfaceRZFourier.from_vmec_input(filename, range="half period", nphi=nphi, ntheta=ntheta)
surf_big = SurfaceRZFourier(dofs=surf.dofs,nfp=nfp, mpol=surf.mpol,ntor=surf.ntor,
                            quadpoints_phi=new_quadpoints_phi,quadpoints_theta=new_quadpoints_theta)
`